docs/cse/records-signals-entities-insights/configure-custom-insight.md

```diff
@@ -50,31 +51,31 @@ This example Custom Insight configuration will generate an Insight as a result o
-1. Enter a description of the Insight, as desired.
-1. For severity, you can choose between a constant severity, or a dynamic severity that is based on the severity of the Signals that trigger the Insight. If you want to configure dynamic severity, skip to the next step. To configure constant severity, select one of: Low, Medium, High, or Critical. 
-1. To configure dynamic severity:
-   1. Choose **dynamic** severity.<br/>  ![dynamic-severity.png](/img/cse/dynamic-severity.png)
-   1. Select a default severity, one of Low, Medium, High, or Critical. 
-   1. **Minimum Signal Severity** and **Insight Severity**. Enter a minimum Signal severity and associated Insight severity value. For example, if you enter 8 and select high, if any Signal in the Insight has a severity of 8 or higher, the custom Insight will have High severity. 
-   1. If desired, you can enter a minimum Signal severity value for other Insight severity levels. For example, you could configure a minimum Signal severity of 4 as the threshold for an Insight severity level of Medium. If you do define multiple thresholds, we honor them from highest to lowest. For example, with the following configuration:
+   2. In the **Type to add a Rule** area, enter a string that the ID of the desired rule contains.
+   3. In the list of rules that appears, scroll to the desired rule and click it.
+   4. If you want to, you can search for and select one or more additional rules.
+   5. If you’ve configured more than one rule, use the **in ... order** clause to specify whether the rules must fire Signals in exact order, or in any order. 
+7. In the **Then Create an Insight** section on the right side of the popup, enter a name for the Insight.
+8. Enter a description of the Insight, as desired.
+9. For severity, you can choose between a constant severity, or a dynamic severity that is based on the severity of the Signals that trigger the Insight. If you want to configure dynamic severity, skip to the next step. To configure constant severity, select one of: Low, Medium, High, or Critical. 
+10. To configure dynamic severity:
+   1. Choose **dynamic** severity.<br/><img src={useBaseUrl('img/cse/dynamic-severity.png')} alt="Dynamic severity" width="300"/>
+   2. Select a default severity, one of Low, Medium, High, or Critical. 
+   3. **Minimum Signal Severity** and **Insight Severity**. Enter a minimum Signal severity and associated Insight severity value. For example, if you enter 8 and select high, if any Signal in the Insight has a severity of 8 or higher, the custom Insight will have High severity. 
+   4. If desired, you can enter a minimum Signal severity value for other Insight severity levels. For example, you could configure a minimum Signal severity of 4 as the threshold for an Insight severity level of Medium. If you do define multiple thresholds, we honor them from highest to lowest. For example, with the following configuration:
```

Comment:
the numbering/indentation is thrown off here in the localhost preview

Reply:
Good catch! Fixed.